README.md

```diff
@@ -44,7 +44,10 @@ Start
     rvm use 2.1.2@freckler
     bundle exec rvmsudo rails server
 
-Copyright
----------
 
-Copyright (c) 2016 [Ombu Labs](http://ombulabs.com)
+License
+-----
+
+Freckler is copyright © 2016 [Ombu Labs](http://ombulabs.com) It is free software, and may be redistributed under the terms specified in the [license].
```

Comment:
@schmierkov I'd just add the link to the `LICENSE` file and skip this paragraph.